content/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-apis-kafka-spark-streaming-and-machine-learning-part-3.md

```diff
@@ -0,0 +1,295 @@
+---
+title: "Fast data processing pipeline for predicting flight delays using Apache
+  APIs: Kafka, Spark Streaming and Machine Learning (part 3)"
+date: 2021-10-11T21:12:59.805Z
+author: Carol McDonald
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-ezmeral-data-fabric
+  - MapR
+---
+```
+"authorDisplayName": "Carol McDonald",
+"tags": "use-cases",
+"publish": "2018-02-08T12:00:00.000Z"
+```
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/data-fabric.html>
+
+**\*Editor's Note:** This is a 3-Part Series, see the previously published posts below:*
```

Comment:
Delete the \ before the Editor's Note:, and remove the asterix at the end of the sentence (just after below:)

Reply:
done